app/views/admin/settings/project_custom_fields/hierarchy/items/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<% html_title t(:label_administration), t("settings.project_attributes.heading"), @custom_field.name, t(:label_item_plural) %>
+
+<%= render(Settings::ProjectCustomFields::EditFormHeaderComponent.new(custom_field: @custom_field, selected: :items)) %>
+
+<%= render(Admin::CustomFields::Hierarchy::ItemsComponent.new(item: @active_item)) %>
```

Comment:
:red_circle: The list of items still partially overlaps the tabbed navigation:

<img width="1011" height="593" alt="image" src="https://github.com/user-attachments/assets/4688d765-ac1f-4afd-9b29-0c43c3b7e75b" />

I'd have expected that we fix this before merging.

(Not quite sure whether this is exactly the right code line to put the comment)